src/js/components/__tests__/__snapshots__/README-test.js.snap

```diff
@@ -8479,6 +8479,14 @@ The id attribute of the input.
 string
 \`\`\`
 
+**maxSize**
+
+Specifies a file size limit for the FileInput
```

Comment:
```suggestion
Specifies a file size limit in bytes for the FileInput
```

Reply:
^^ Updates to docs should be filed on grommet-site now, but leaving this comment here as a suggestion to the wording